docs/pages/docs/reference/command-line-reference.mdx

```diff
@@ -75,6 +75,18 @@ Run NPM scripts across all packages in specified scope. Tasks must be specified
 to the tasks to be executed. Note that these additional arguments will _not_ be passed to
 any additional tasks that are run due to dependencies from the [pipeline](/docs/reference/configuration#pipeline) configuration.
 
+Uses glob patterns via [`doublestar`](https://github.com/bmatcuk/doublestar) under the hood. Specifying multiple glob patterns adds to the result.
+
+##### doublestar globbing patterns
+
+Just a quick overview.
+
+- `*` matches any number of characters, but not `:`
```

Comment:
What would this look like if we were not special-casing `:`? I'm a little wary of introducing two kinds of globs in the UX.